tuxemon/state.py

```diff
@@ -85,24 +85,17 @@ class should be created. Update must be overloaded in the child class.
     transparent = False  # ignore all background/borders
     force_draw = False  # draw even if completely under another state
 
-    def __init__(self, parent: StateManager) -> None:
+    def __init__(self, **_: Any) -> None:
```

Comment:
What happened with the parent parameter?

Reply:
This attribute is only used by one class, so I changed that class to require it to be passed in. Otherwise all states would require it to be passed in during pushstate (which would be annoying), or we'd have to do something weird.